src/SCREENS.js

```diff
@@ -6,6 +6,7 @@ export default {
     HOME: 'Home',
     LOADING: 'Loading',
     REPORT: 'Report',
+    LOG_IN_WITH_SHORT_LIVED_TOKEN: 'LogInWithShortLivedToken',
```

Comment:
Can we align `SHORTLIVED` and `SHORT_LIVED` to help make this searchable?

Reply:
Oh true. Yes, I will once I figure out which PR we should be using.